contrib/opencensus-ext-azure/opencensus/ext/azure/metrics_exporter/standard_metrics.py

```diff
@@ -75,6 +77,35 @@ def get_process_private_bytes_metric():
     return gauge
 
 
+def get_processor_time():
+    cpu_times_percent = psutil.cpu_times_percent()
+    return 100 - cpu_times_percent.idle
+
+
+def get_processor_time_metric():
+    """ Returns a derived gauge for the processor time.
+
+    Processor time is defined as a float representing the current system
+    wide CPU utilization minus idle CPU time as a percentage. Idle CPU
```

Comment:
It'll be good to clarify the actual range of this value (e.g. **[0.0, 100.0]**).